Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/GMUGeometryRendererTest.swift

```diff
@@ -0,0 +1,281 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import GoogleMapsUtils
+
+final class GMSMapViewMock: GMSMapView {
+    var addedOverlays: [GMSOverlay] = []
+    var clearCalled = false
+
+    func add(_ overlay: GMSOverlay) {
+        addedOverlays.append(overlay)
+    }
+
+    override func clear() {
+        clearCalled = true
+        addedOverlays.removeAll()
+    }
+}
```

Comment:
[self review] Code will be refactored post the Google Maps iOS framework issue is resolved. More details in the PR description.